Core/Libraries/Source/WWVegas/WWLib/stringex.h

```diff
@@ -182,3 +191,141 @@ template<size_t Size> size_t strlcat_t(char (&dst)[Size], const char *src) { ret
 template<size_t Size> size_t strlmove_t(char (&dst)[Size], const char *src) { return strlmove_t(dst, src, Size); }
 template<size_t Size> size_t strlmcat_t(char (&dst)[Size], const char *src) { return strlmcat_t(dst, src, Size); }
 #endif
+
+// Templated strncmp.
+// Compares up to maxcount chars or a null byte is encountered, whichever comes first.
+// Returns < 0 if str1 is less than str2, 0 is str1 and str2 are equal and > 0 if str2 is greater than str1.
+template<typename CharT> int strncmp_t(const CharT *str1, const CharT *str2, const size_t maxcount)
+{
+	if (str1 == nullptr || str2 == nullptr)
```

Comment:
This test seems not necessary:

https://en.cppreference.com/w/c/string/byte/strncmp

> The behavior is undefined when either lhs or rhs is the null pointer.